src/components/typeahead/typeahead.component.ts

```diff
@@ -1,10 +1,24 @@
-import { FocusOrigin } from '@angular/cdk/a11y';
-import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, HostBinding, HostListener, Input, OnChanges, OnDestroy, Output, SimpleChanges, TemplateRef } from '@angular/core';
-import { BehaviorSubject, combineLatest, Subject } from 'rxjs';
-import { distinctUntilChanged, takeUntil } from 'rxjs/operators';
-import { InfiniteScrollLoadedEvent, InfiniteScrollLoadFunction } from '../../directives/infinite-scroll/index';
-import { TypeaheadOptionEvent } from './typeahead-event';
-import { TypeaheadService } from './typeahead.service';
+import {FocusOrigin} from '@angular/cdk/a11y';

```

Comment:
Can we fix the formatting, all our scripts have spaces around the imports eg: `{ FocusOrigin }` instead of `{FocusOrigin}`

Reply:
Done